Quote Generator/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quote Generator</title>
+
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@700&family=Lobster+Two:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+
+    <!-- font awesome cdn -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div>
+    <h1>Quote Generator</h1>
+    <div class="container">
+        <i class="fas fa-quote-left"></i>
+            <div  id="quotes">
+                Everything in life is luck.
+            </div>
+            <hr>
+            <div id="author">
+                By Donald Trump
+            </div>
+            <button onclick="getQuotes()">New Quotes</button>
+        
+    </div>
+    
+</div>
+    
+
+    <script src="script.js">
+       
+    </script>
+</body>
+</html>
```

Comment:
Add a new line at the end of the file..

Use HTML beautifier https://codebeautify.org/htmlviewer